.circleci/config.yml

```diff
@@ -48,3 +87,16 @@ workflows:
           matrix:
             parameters:
               eslint-version: *supported-eslint-versions
+              
+  build_and_test_and_deploy:
```

Comment:
We can do that, but we'd lose the accurate and descriptive workflow name (build_and_test vs build_and_test_and_deploy) since afaik there's no way to change the name based on the jobs run.

Given the low overhead, I feel it'd be clearer and less confusing if devs can at a glance see what's happening with the CI from Github. Otherwise, with the circleci suggested naming schemes, we'd have a `build_and_test` that _sometimes_ deploys, or a `build_and_test_and_deploy` that _mostly_ doesn't deploy.